pkg/pipeline/crd.go

```diff
@@ -79,12 +83,30 @@ func (cg *CRDGenerator) Generate(cfg *config.Resource) error {
 	if err != nil {
 		return errors.Wrap(err, "cannot print the type list")
 	}
+	parametersType := ""
+	observationType := ""
+	for _, t := range typeList {
+		if reParametersType.MatchString(strings.TrimPrefix(t.Obj().Name(), cfg.Kind)) {
```

Comment:
`buildResource` already [returns](https://github.com/crossplane-contrib/terrajet/blob/6d7fb7e/pkg/types/builder.go#L57) the parameter and observation types but we just ignore them at the highest level call. Is there a way to bring that information here instead of doing a search in the list?

Reply:
Great idea, done!